.go-version

```diff
@@ -1 +1 @@
-1.70.0
\ No newline at end of file
+1.70.2
```

Comment:
`1.70.1`?

Reply:
I figured patch-level version numbers are free and this doesn't really make a difference :)